### DISCUSSION TITLE
January 16–20 2023

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#137] [Progress Report]
🚴🏻 **Progress**

- Begun work on #135 

😖 **Regressions**

None

💬 **To Discuss**

None

📆 **Next Week**

- #135

**Comment by ian-noaa** [comment]
🚴🏻 **Progress**

- Working with VLab to deploy the Lambda function for the Data Processing Pipeline
- Working with VLab to version control and automate the production Kubernetes config

😖 **Regressions**

None

💬 **To Discuss**

None

📆 **Next Week**

- Continue working with VLab
